src/wp-includes/rest-api/endpoints/class-wp-rest-site-health-controller.php

```diff
@@ -0,0 +1,244 @@
+<?php
+/**
+ * REST API: WP_REST_Site_Health_Controller class
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 5.5.0
+ */
+
+/**
+ * Core class for interacting with Site Health elements.
+ *
+ * @since 5.5.0
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Site_Health_Controller extends WP_REST_Controller {
+
+	private $site_health;
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
```

Comment:
Can we pass the `WP_Site_Health` instance to the constructor here? That way we can make this testable.